.github/workflows/main.yml

```diff
@@ -265,27 +265,35 @@ jobs:
       matrix:
         include:
           # PYTHON CHECKS ON CURRENT FEDORA AND PYTHON TOOLS VERSION
+          - dependencies: >
+              pylint
+              python3-requests
+              python3-pygithub
```

Comment:
This test is probably failing because `python3-semantic_version` needs to be added as a dependency.

Reply:
Whoops! I thought all the tests had passed...

Reply:
The first series of tests passed, prior to when the second commit added.  But it passes now.